### PR TITLE
[ENH] Despaz bluetooth display

### PR DESCRIPTION
**Pull request by rksh**
not perfect, but it makes the display less unreliable. prep for route logging/display with density.
also contains a small feature-detection fix.

### CODE REVIEW

File: wiglewifiwardriving/src/main/java/net/wigle/wigleandroid/listener/BluetoothReceiver.java [hunk 1]
```diff
@@ -140,13 +147,19 @@
     private long adUuidNoScanUuid = 0;
     private long scanUuidNoAdUuid = 0;
 
+    // prev/current sets of BSSIDs for each scan. ~ redundant w/ sets in SetBackedNetworkList in current-only mode...
+    private Set<String> latestBt = new HashSet<>();
+    private Set<String> prevBt = new HashSet<>();
 
-    public BluetoothReceiver(final MainActivity mainActivity, final DatabaseHelper dbHelper ) {
+    private Set<String> latestBtle = new HashSet<>();
+    private Set<String> prevBtle = new HashSet<>();
```
Comment (bobzilladev) [inline wiglewifiwardriving/src/main/java/net/wigle/wigleandroid/listener/BluetoothReceiver.java]: are we sure these will only be touched by one thread?

Comment (rksh) [reply wiglewifiwardriving/src/main/java/net/wigle/wigleandroid/listener/BluetoothReceiver.java]: I'm not perfectly sure the bt le scanner will always write from the same thread. the .size() calls could result in some concurrency risk. I'll change the implementation.

Comment (rksh) [reply wiglewifiwardriving/src/main/java/net/wigle/wigleandroid/listener/BluetoothReceiver.java]: should be safe now

Comment (bobzilladev) [reply wiglewifiwardriving/src/main/java/net/wigle/wigleandroid/listener/BluetoothReceiver.java]: the i* vars aren't used elsewhere, I'd just move the new hashset into the next line so no one can get a handle on the non-synchronized reference.

**Pull request merged**